contents/handbook/people/onboarding.md

```diff
@@ -157,3 +157,17 @@ We use GitHub Projects to track the status of Issues in an easily viewable way.
 You can also assign an Issue to a specific person, and tag it with a relevant label - use these to help people filter more easily.
 
 Each Small Team has its own Project for tracking their Issues - [full list here](https://github.com/orgs/PostHog/projects). Most teams run two week sprints - as part of onboarding, you will be invited to the relevant planning meetings. 
+
+## 60 & 90 day Check-ins
```

Comment:
Where this differs to my own personal approach so far is that I normally bookmark 30,60 and 90 day.

* **30 day**
  * Chance to give initial feedback about how things are going, especially if there is anything concerning (lack of shipping, communication etc.) and to reinforce positive work so that the person feels they're on the right track
* **60 day**
  * If things are going well this is the right time to indicate that (relaxes fears of the "looming" end of probation
  * If things are not going well this is the chance to make a point of it and speak clearly about what the expectations are and what can be changed (this should hopefully not be a complete shock as the 30 day check-in will have already potentially caught some things
* **90 day**
  * If it gets to this point nothing should be too surprising - if feedback was positive at the 60 day point then likely it all works out
  * If the feedback at either past point was negative then the person hopefully was leaning into correcting these things and proving that they are the right person for the job. It shouldn't then be a total surprise if the decision is to part ways.

Maybe its not too important but to me the earlier there is a checkpoint to raise red (or orange) flags, the better.